Doc/library/enum.rst

```diff
@@ -667,28 +667,28 @@ It is also possible to name the combinations::
     ...     X = 1
     ...     RWX = 7
     >>> Perm.RWX
-    <Perm.RWX: 7>
+    Perm.RWX
     >>> ~Perm.RWX
-    <Perm.-8: -8>
+    Perm.-8
 
 Another important difference between :class:`IntFlag` and :class:`Enum` is that
 if no flags are set (the value is 0), its boolean evaluation is :data:`False`::
 
     >>> Perm.R & Perm.X
-    <Perm.0: 0>
+    Perm.0
```

Comment:
`Perm(0)` would look better.

Reply:
I went with `0x0` to match other non-Flag values, similarly to the way `re.RegexFlag` does it.